docs/source/getting-started/tabular-data/grouping.ipynb

```diff
@@ -614,9 +700,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Public Group Sizes\n",
+    "## Invariant Group Sizes\n",
     "\n",
-    "It is also possible to declare partition sizes as public information.\n",
+    "It is also possible to declare partition sizes as data invariants.\n",
```

Comment:
FWIW, unless it just wasn't accurate, I think the former description was more clear.